polyfills/WeakSet/config.json

```diff
@@ -3,12 +3,13 @@
 		"es6"
 	],
 	"browsers": {
-		"ie": "7 - *",
-		"ie_mob": "10 - *",
-		"safari": "*",
-		"chrome": "<38",
+		"ie": "7 - 11",
+		"ie_mob": "10 - 11",
+		"safari": "<9",
+		"chrome": "<36",
 		"firefox": "<34",
-		"ios_saf": "*",
+		"android": "<5",
```

Comment:
android 5.1 does not have full WeakSet support, it is missing 3 features:
- iterator closing
- no `WeakSet.prototype.clear` method
- `.has` and `.delete` methods accept primitives